### DISCUSSION TITLE
Debugging infeasible models

### DISCUSSION THREAD

**Discussion by zoharl3** [yalmip/YALMIP#969] [Possible bug or missing functionality] [answered]
There's this comprehensive page:

https://yalmip.github.io/debugginginfeasible

Usually, that's what I do more or less.

I just wanted to point out that it's possible to write a general tool to find the problematic constraint given a problem:
- Change the objective to 1.
- Run the problem with all possibilities of constraints turned on and off. Offer also a faster option that turns on constraints inclemently and reports the first failure.
- Maybe name the constraints automatically as the line number in code.

**Comment by johanlofberg** [comment, marked as answer]
Yes, it's something that has been on the todo-list forever (using bisection though, locking at all combinations is combinatorially completely out of the picture, automatic line-tagging is not possible)
